src/routes/chat-with-tools/+page.svelte

```diff
@@ -0,0 +1,57 @@
+<script lang="ts">
+  import { useChat } from '@ai-sdk/svelte';
+
+  const { input, handleSubmit, messages, addToolResult } = useChat({
+    api: '/api/use-chat-tools',
+    maxToolRoundtrips: 5,
+    // run client-side tools that are automatically executed:
+
+    async onToolCall({ toolCall }) {
```

Comment:
?

Reply:
we'll remove all unnecessary parts in next commit.